src/AppBundle/PullRequests/Repository.php

```diff
@@ -169,6 +171,47 @@ public function removeCommentsIfExists(PullRequest $pullRequest, $pattern, $user
         return false;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getMergedFromWithCommentsFrom(string $mergedFrom, string $commentedBy, ?DateTime $since = null)
+    {
+        if (null === $since) {
+            $since = (new DateTime())->sub(DateInterval::createFromDateString('30 days'));
+        }
+        $query = 'repo:'.$this->repositoryOwner.'/'.$this->repositoryName.' is:pr is:merged author:'.$mergedFrom.' commenter:'.$commentedBy.' merged:>='.$since->format('Y-m-d');
+        $search = '{
```

Comment:
This kind of string should be store in a file :thinking:  
(Can be done in a new PR)